cinema/tests.py

```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
+
+# Create your tests here.
```

Comment:
Consider adding test cases for your models, views, and serializers. Testing is crucial for ensuring the reliability and correctness of your application. You can start by creating test cases for the `Movie` model and the `MovieSerializer`.